_posts/jasonwang/2016-04-29-jasonwang-open-source.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+author: wagerpascal
+title: "Jason's Open Source Contribution"
+---
+
+To be honest, it was pretty tough to find an open source project that fit my interests, was mainly in python, and was overall easy enough for me to understand.
+
+I took a peek through the Github trending open source projects (under the Python listing) and kept reading through (maybe 15-20 different projects) until I found https://github.com/YongHaoWu/NeteaseCloudMusicFlac . It was a pretty simple program, which only required the one python file to crawl through Baidu music players and downloading the resulting FLAC file from the site. Pretty cool small project, and I was really glad that I’m semi-literate in Chinese to the point that I understood the gist of what the guy wrote. I found the TODO list on the main page, and read through it (Rough translation here for convenience).
+
+1. Reorganize into PEP8 format.
```

Comment:
A note: there are programs (or features within code editors) called **linters** that can automatically enforce PEP-8 and other code styles.  Just FYI.